components/FetchedDogs/FetchedDogs.tsx

```diff
@@ -1,119 +1,86 @@
 import React, { useState, useEffect, useRef, useCallback } from "react";
-import { useLocalStorage } from "../../hooks/useLocalStorage";
+import { useFetchDog } from "../../hooks/useFetchDog";
 
-import {
-  DogDetails,
-  DogsDetails,
-  likedDogsDetails,
-  useAppContext,
-} from "../../context";
+import { likedDogsDetails, useAppContext } from "../../context";
 
 import { HeartButton } from "../Buttons/HeartButton/HeartButton";
 import { LoadingSpinner } from "../LoadingSpinner/LoadingSpinner";
 
 import style from "./FetchedDogs.module.css";
 import { Breed } from "../../data/breedsData";
-
-type fetchedDog = {
-  message: string;
-  status: string;
-};
+import Link from "next/link";
 
 export const FetchedDogs = () => {
   const {
+    isMosaic,
     loading,
     setLoading,
     fetchedDogs,
     setFetchedDogs,
     likedDogs,
     setLikedDogs,
     setIsError,
-    apiCallCounter,
-    setApiCallCounter,
+    storagedBreeds,
   } = useAppContext();
 
   const [breed, setBreed] = useState<Breed>();
   const [random, setRandom] = useState(0);
```

Comment:
What does it mean?

Reply:
Not sure what's this question is pointing. Random state? If yes that was an old idea to keep random breed index tracking. Not necessary any more. I change it.

Reply:
What name means, because `random` tells nothing. After what you wrote it should be more like `randomBreedIndex`